docs/tips-n-tricks/Wormholes/gas-harvesting/ninja-huffing.md

```diff
@@ -18,15 +18,15 @@ Only one site cannot be ninja-huffed : the [Ordinary Perimeter Reservoir](https:
 
 ### Ships
 
-All ships and fits can ninja-huff
+All ships and fits can ninja-huff, as it requires no specific gear.
 
 ## How to
 
-Whenever a player first warps to a site (gas site, ore belts, etc), a timer is triggered, at the end of which rats will spawn.
+Whenever a player first warps to a site (gas site, ore belts, etc) **and is uncloaked**, a timer is triggered, at the end of which rats will spawn.
```

Comment:
Are you sure about that ? I didn't know myself

Reply:
https://wiki.eveuniversity.org/Ghost_Sites#Getting_Loot

I might be getting confused with the hacking sites not activating until you decloak. I'm looking but can't find anything similar specifically about Gas Sites.

Reply:
Lets leave this out for now then. I'll test it out soon.

Reply:
I did check, and warping in cloaked *will* trigger the rats spawn timer.